### DISCUSSION TITLE
Why does the request body JSON reload after sending it?

### DISCUSSION THREAD

**Discussion by HubKing** [Kong/insomnia#8268] [Help]
For example, if I were seeing the 45th line of the request body JSON, if I send it and the response is received, the body JSON view scrolls back to the top, probably because it was reloaded.

Is this a bug? I find it very annoying.

[Screencast_20241225_002633.webm](https://github.com/user-attachments/assets/fd724db6-d22b-428a-86b4-9f9eb059e9ec)

**Comment by poojithap11** [comment]
Hi,

Could you try testing this in latest version and let us know if you're still facing the same issue.

Thank you.

**Comment by poojithap11** [comment]
Hi,

Did you get a chance to test testing this in latest version and let us know if you're still facing the same issue.

Thank you.

**Comment by HubKing** [comment]
This has not been happening for a while, so I think it can be closed. Thank you.
